lib/ansible/plugins/callback/default.py

```diff
@@ -350,8 +350,9 @@ def v2_runner_item_on_skipped(self, result):
 
     def v2_playbook_on_include(self, included_file):
         msg = 'included: %s for %s' % (included_file._filename, ", ".join([h.name for h in included_file._hosts]))
-        if 'item' in included_file._args:
-            msg += " => (item=%s)" % (self._get_item_label(included_file._args),)
+        label = self._get_item_label(included_file._vars)
```

Comment:
@Yadnyawalkya Thanks for the PR. Could you please add a test [here](https://github.com/ansible/ansible/blob/6fa070e824ab4bcd694f78edba5ff2261a6384e8/test/integration/targets/callback_default/runme.sh#L178) and a changelog entry for this change [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments)? Thanks.